test/server/controllers/categories.test.ts

```diff
@@ -0,0 +1,52 @@
+import { expect } from 'chai'
+import { suite, test } from 'mocha-typescript'
+import * as supertest from 'supertest'
+import * as _ from 'lodash'
+import { logInUser } from './helpers'
+import { UserAttributes } from '../../../src/server/models'
+import { app } from '../../../src/server'
+import { factory, createOffer, createCategory } from '../factory'
+import { logger } from '../../../src/server/services'
+
+const request = supertest(app)
+const createUser: UserAttributes = {
```

Comment:
We should use the createUser from the factory, or consider moving it to the factory